packages/compass-connections-navigation/src/base-navigation-item.tsx

```diff
@@ -59,10 +57,8 @@ const itemWrapperStyles = css({
   display: 'flex',
   height: ROW_HEIGHT,
   alignItems: 'center',
-  paddingLeft: spacing[100],
-  paddingRight: spacing[100],
+  paddingRight: spacing[400],
```

Comment:
I actually think we can push this further right. Usually there's a scrollbar which brings it out of alignment with the three dot menu next to CONNECTIONS anyway.

Reply:
Further right sounds good.